<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationScanWidget</class>
 <widget class="QWidget" name="ConfigurationScanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigurationScanWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>扫描以下内存类型:</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="EnableMemPrivate">
          <property name="toolTip">
           <string>私有内存</string>
          </property>
          <property name="text">
           <string>MEM_PRIVATE</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="EnableMemImage">
          <property name="toolTip">
           <string>映射与可执行的镜像文件片段(加载到内存的EXE或者DLL等可执行文件)</string>
          </property>
          <property name="statusTip">
           <string>映射与可执行的镜像文件片段(加载到内存的EXE或者DLL等可执行文件)</string>
          </property>
          <property name="text">
           <string>MEM_IMAGE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="EnableMemMapped">
          <property name="toolTip">
           <string>映射与不可执行的镜像文件片段(例如:页文件,模拟器内存)</string>
          </property>
          <property name="text">
           <string>MEM_MAPPED(慢)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>&quot;所有类型&quot;包括:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="IncludeByte">
          <property name="text">
           <string>字节</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Include2Byte">
          <property name="text">
           <string>2字节</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Include4Byte">
          <property name="text">
           <string>4字节</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Include8Byte">
          <property name="text">
           <string>8字节</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="IncludeFloat">
          <property name="text">
           <string>单浮点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="IncludeDouble">
          <property name="text">
           <string>双浮点</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>高级</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="EnablePageWriteCombine">
          <property name="toolTip">
           <string>PAGE_WRITECOMBINE</string>
          </property>
          <property name="statusTip">
           <string>PAGE_WRITECOMBINE</string>
          </property>
          <property name="text">
           <string>扫描受写组合保护内存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="EnablePageGuard">
          <property name="toolTip">
           <string>PAGE_GUARD</string>
          </property>
          <property name="statusTip">
           <string>PAGE_GUARD</string>
          </property>
          <property name="text">
           <string>扫描受保护内存</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>224</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
